projects/project-2/readings/Project 2 - SPARQL library

```diff
@@ -222,6 +274,10 @@ This query evaluates properties employed by instances of multiple classes, revea
 
 14. Detecting Inconsistent Property Ranges Across Subclass Hierarchies (Level 3 - 20 points)
 
+Title: Inconsistent Subclass Property Range
+Constraint Description: Detects inconsistencies in property ranges within class hierarchies.
+Severity: Error
+
 PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
 PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 PREFIX owl: <http://www.w3.org/2002/07/owl#>
```

Comment:
This is conceptually fine, but I think you should add some details, like the class being an owl:class and the range being connected through rdfs:range

Reply:
I agree!